backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterWithKeyspaceCreatedFactory.java

```diff
@@ -54,16 +54,11 @@ public Configuration replicationFactor(int replicationFactor) {
         }
         
         public Cluster clusterWithInitializedKeyspace() {
-            if (isKeyspacePresent(cluster, keyspace)) {
-                createKeyspace(cluster, keyspace, replicationFactor, durableWrites);
-            }
+            createKeyspace(cluster, keyspace, replicationFactor, durableWrites);
```

Comment:
how this change is related to this PR ?

Reply:
The boot time optimization of looking if the keyspace exist before creating it is pointless now that we recycle keyspace.

Furthermore a "IF NOT EXISTS" is used while creating the keyspace.